src/views/CategoryView.vue

```diff
@@ -1,7 +1,12 @@
 <template>
-  <div>
-    <QuestionsList />
-  </div>
+  <Suspense>
+    <template #default>
+      <QuestionsList />
+    </template>
+    <template #fallback>
+      <SpinnerLoader />
```

Comment:
Loader component should have `visible` flag like props.
Make it on center
![Screenshot 2022-11-28 at 11 09 54](https://user-images.githubusercontent.com/10779650/204226158-9320ccb1-2d03-41c9-bae4-8ec56610715c.jpg)